src/gui/interface/dialog.py

```diff
@@ -9,7 +10,7 @@
 import textwrap
 
 
-class TextBox(pygame.sprite.Sprite):
+class TextBox(Sprite):
```

Comment:
Uh, why did you make it inherit from Sprite ?

Reply:
I thought that it would make things a little easier. For example it already has the hitbox_rect attribute, which is now necessary for rendering items on screen. In addition, it is now possible to easily extend the drawing function if we may need it in the future.